styles/GlobalStyles.js

```diff
@@ -25,11 +25,11 @@ const GlobalStyles = createGlobalStyle`
     width: 100%;
     font-family: 'Miners', sans-serif;
 
-    @media (${breakpoints.tablet}) {
+    @media (${tablet}) {
```

Comment:
Actually when I come to think on it. There is no point of changing font size in relation to width of the screen